conduit-axum/src/tests.rs

```diff
@@ -36,20 +40,7 @@ impl Handler for Panic {
 struct InvalidHeader;
 impl Handler for InvalidHeader {
     fn call(&self, _req: &mut ConduitRequest) -> HandlerResult {
-        Response::builder()
-            .header("invalid-value", "\r\n")
-            .body(Bytes::from_static(b"discarded"))
-            .map_err(box_error)
-    }
-}
-
-struct InvalidStatus;
-impl Handler for InvalidStatus {
-    fn call(&self, _req: &mut ConduitRequest) -> HandlerResult {
-        Response::builder()
-            .status(1000)
```

Comment:
this case appears to be impossible (?) to reproduce with `axum` responses, but since it's that hard we probably don't need it that much anyway.